packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -239,8 +224,9 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
           const metadata = await IdentityMetadataWrapper.fetchFromURL(metadataURL)
           const attestationServiceURLClaim = metadata.findClaim(ClaimTypes.ATTESTATION_SERVICE_URL)
 
-          if (attestationServiceURLClaim === undefined) {
-            throw new Error(`No attestation service URL registered for ${issuer}`)
+          if (!attestationServiceURLClaim) {
+            console.info(`No attestation service URL registered for ${issuer}`)
```

Comment:
FYI in https://github.com/celo-org/celo-monorepo/pull/2105/files I'm removing the logging of the errors to the console. IMO it should be the responsibility of the consumer of contractkit to determine appropriate logging when running the code

Reply:
So would this throw? just return null?

Reply:
We catch it in the surrounding try, and thus just return null. In effect, the method still returns the actionable attestations. If a consumer wanted to log that some of the attestations "should" be completable, but there was not attestation service registered (or any other issue), consumer should use the new method of `getNonCompliantIssuers`